### DISCUSSION TITLE
Github Education Benefits Still Not Avaible

### DISCUSSION THREAD

**Discussion by MriMouse** [community/community#142667] [GitHub Education]
### Hi everyone,

I haven't received the benefits of an education account yet, and I need it urgently in the next two days. Can the staff help me? If this works, I'd be very grateful!

**Comment by MriMouse** [comment]
😢😢😢

**Comment by davevad93** [comment]
Hi @MriMouse ,

GitHub no longer provides manual reviews, the waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @MriMouse was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Reply by MriMouse** [reply]
thanks😢，hope I can use it "soon"

**Reply by davevad93** [reply]
You're welcome @MriMouse , don't worry you'll get your benefits soon! 👍
